crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI063.py

```diff
@@ -0,0 +1,64 @@
+# See https://peps.python.org/pep-0484/#positional-only-arguments
+# for the full details on which arguments using the older syntax should/shouldn't
+# be considered positional-only arguments by type checkers.
+from typing import Self
+
+def bad(__x: int) -> None: ...  # PYI063
+def also_bad(__x: int, __y: str) -> None: ...  # PYI063
+def still_bad(__x_: int) -> None: ...  # PYI063
+
+def no_args() -> None: ...
+def okay(__x__: int) -> None: ...
+# The first argument isn't positional-only, so logically the second can't be either:
+def also_okay(x: int, __y: str) -> None: ...
+def fine(x: bytes, /) -> None: ...
```

Comment:
Can I suggest the test case
```python
def still_fine(_x: int) -> None: ...
```
This should be fine right?

Reply:
seems good, I'll add it.